pkg/meta/version_service.go

```diff
@@ -44,6 +45,20 @@ else
 	redis.call("hset", KEYS[1], ARGV[1], ARGV[2]);
 	return 1
 end`)
+	luaDelKeysWithPrefix = redis.NewScript(`local cursor = tonumber(ARGV[1]) or 0
+local pattern = ARGV[2]
+local count = tonumber(ARGV[3]) or 1000
+local field = KEYS[1]
+
+local scan_result = redis.call('SCAN', cursor, 'MATCH', pattern, 'COUNT', count)
+local new_cursor = tonumber(scan_result[1])
+local keys = scan_result[2]
+
+for _, key in ipairs(keys) do
+    redis.call('HDEL', key, field)
+end
+
+return new_cursor`)
```

Comment:
I'd move lua script to constants for the readability.

Reply:
i think that it is readable since it is only wrapped with `redis.NewScript()`. Otherwise there will be a set of consts and set of vars with similar names. Or we will have to call `redis.NewScript()` on each script invocation. But in this case we will be calculating script hash every time.

Reply:
> Otherwise there will be a set of consts and set of vars with similar names

I'd actually prefer this option.

Reply:
Also, one of the options can be to use separate files for lua scripts, like

```go
import _ "embed"

//go:embed script.lua
var luaScript string
```

Reply:
> I'd actually prefer this option. Anyways, compiler would do the same.

I'd like to keep current approach for several reasons:
- it is not useful to have named constant for string with exactly one usage
- when it may increase readabily it will multiply optoins in code completion
- we will have to change scripts in all files to have the same style across the project 
- [go-redis examples](https://github.com/redis/go-redis/blob/master/example/lua-scripting/main.go) and projects like asynq also using single vars.

> Also, one of the options can be to use separate files for lua scripts, like

i like this option because it enables syntax highlight but i don't know if it is worth it to have a separate file for each script.